### PR TITLE
documenting the rollout process in README. clean up of other files.

### PR DESCRIPTION
**Pull request by muellerk22**
## Description of the change

> Did something awesome w/out a breaking change.

## Type of change
- [ ] Bug fix
- [X ] New feature

### Development
- [ ] All changed code has 80% unit test coverage
- [X ] All changed code has been automatically (smoke test or otherwise) or manually verified in `alfa` (or with a cross namespace setup, e.g. developer namespace for this feature, pointing at shared `alfa` resources)

### Code review 
- [ ]  This pull request has a title that includes the ticket # and a short useful summary, e.g. `CP-4051: Create TEMPLATE Feature Repo`.

### CODE REVIEW

File: helm-charts/test/cwagent/cwagent-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# create configmap for cwagent config
+apiVersion: v1
+data:
+  # Configuration is in Json format. No matter what configure change you make,
+  # please keep the Json blob valid.
+  cwagentconfig.json: |
+    {
+      "agent": {
+        "debug": true
```
Comment (silvexis) [inline helm-charts/test/cwagent/cwagent-configmap.yaml:9]: disable debug

File: helm-charts/test/cwagent/cwagent-daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    spec:
+      containers:
+        - name: cloudwatch-agent
+          # below is the sample version
+          #image: amazon/cloudwatch-agent:1.247348.0b251302
+          # below is the latest release
+          #image: amazon/cloudwatch-agent:1.247349.0b251399
+          # Below 1.247347.4
+          #image: amazon/cloudwatch-agent:1.247347.4b250525
+          # Below is the Pellaton version
```
Comment (silvexis) [inline helm-charts/test/cwagent/cwagent-daemonset.yaml:24]: remove these commented out versions

File: helm-charts/test/cwagent/cwagent-configmap.yaml (hunk 1 above)
Comment (muellerk22) [reply helm-charts/test/cwagent/cwagent-configmap.yaml:9]: will do.

File: helm-charts/test/cwagent/cwagent-rollout-daemonset.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      labels:
+        name: cloudwatch-agent
+    spec:
+      containers:
+        - name: cloudwatch-agent
+          # below is the sampel verion
+          #image: amazon/cloudwatch-agent:1.247348.0b251302
+          # below is the latest release
+          #image: amazon/cloudwatch-agent:1.247349.0b251399
+          # Below is the Pellaton version
```
Comment (silvexis) [inline helm-charts/test/cwagent/cwagent-rollout-daemonset.yaml:26]: remove comments

**Review by silvexis — CHANGES_REQUESTED** [review]
few minor changes

**Review by silvexis — APPROVED** [review]

**Pull request merged**